source/solvers/navier_stokes_base.cc

```diff
@@ -871,6 +869,33 @@ NavierStokesBase<dim, VectorType, DofsType>::postprocess(bool firstIter)
         }
     }
 
+  if (this->nsparam.post_processing.calculate_average_velocities)
+    {
+      double epsilon = 1e-6;
```

Comment:
Epsilon should be declared just straight above where it is used.

Reply:
It is used in the else if and this is right before the if, I can not really declare closer where it is use. In this case, don't you think it's better to use 1e-6 instead of epsilon?